tests/testthat/test-Renderer.R

```diff
@@ -1,33 +1,33 @@
 testthat::test_that("Renderer object can be created", {
-  testthat::expect_error(Renderer$new(), regexp = NA)
+  testthat::expect_error(teal.reporter::Renderer$new(), regexp = NA)
```

Comment:
you shouldn't need to use teal.reporter:: in the tests ?? See e.g. https://github.com/insightsengineering/teal.slice/blob/b0c6dafb0410371fa54b7db13b962fdbabccc6d3/tests/testthat/test-FilteredData.R#L6

Reply:
I think it is a good practice

Reply:
I'll leave this one to you